src/main/java/by/yahor_kulesh/App.java

```diff
@@ -1,46 +1,15 @@
 package by.yahor_kulesh;
 
-import by.yahor_kulesh.model.Ticket;
+import by.yahor_kulesh.model.Data;
+
 import by.yahor_kulesh.services.TicketService;
-import by.yahor_kulesh.validators.InputValidator;
 
 
-public class App {
+
+public class App extends Data {
 
     public static void main(String[] args) {
-        while(true) {
-            System.out.println("""
-                        Welcome to the Ticket Service!
-                        1.Create empty Ticket
-                        2.Create limited ticket([Concert hall], [event code] and [time] required)
-                        3.Create full ticket
-                        0.Exit
-                        """
-            );
-            switch (InputValidator.inputInt()){
-                case 1:{
-                    System.out.println(new Ticket());
-                    break;
-                }
-                case 2: {
-                    System.out.println(TicketService.createLimitedTicket());
-                    break;
-                }
-                case 3: {
-                    System.out.println(TicketService.createFullTicket());
-                    break;
-                }
-                case 0: {
-                    return;
-                }
-                default: {
-                    System.out.println("Try again!");
-                }
-            }
-        }
+	    TicketService.applicationMethod();
```

Comment:
"applicationMethod" - is too general name. It is not clear what exactly will be done when we call it. If it is void and returns nothing, it has to be named in verb form.